lib/datasource/pypi/index.spec.ts

```diff
@@ -304,6 +307,23 @@ describe('datasource/pypi/index', () => {
       });
       expect(res.isPrivate).toBeTrue();
     });
+    it('process data from simple endpoint with hyphens', async () => {
+      httpMock
+        .scope('https://pypi.org/simple/')
+        .get('/package-with-hyphens/')
+        .reply(200, hyphensResponse);
+      const config = {
+        registryUrls: ['https://pypi.org/simple/'],
+      };
+      expect(
+        await getPkgReleases({
+          datasource,
+          ...config,
+          depName: 'package--with-hyphens',
+        })
+      ).toMatchSnapshot();
```

Comment:
Please add partial response object here to have a more precise tests

Reply:
Is it better now, did you mean something like that?